manifests/composer/auto_update.pp

```diff
@@ -25,16 +29,22 @@
   $max_age,
   $source,
   $path,
-  $environment = undef,
+  $proxy_type   = undef,
+  $proxy_server = undef,
 ) {
 
   if $caller_module_name != $module_name {
     warning('php::composer::auto_update is private')
   }
 
+  $env = $proxy_type? {
+    undef   => {},
```

Comment:
It appears this isn't puppet 3 compatible. From https://docs.puppet.com/puppet/3.8/reference/lang_conditional.html#values

> Values may be any of the following:

> Any literal value, with the exception of hash literals